bundles/binding/org.openhab.binding.zibase/src/main/java/org/openhab/binding/zibase/internal/ZibaseGenericBindingProvider.java

```diff
@@ -49,6 +49,11 @@
     static final HashMap<String, Vector<String>> itemIdNameMap = new HashMap<String, Vector<String>>();
 
     /**
+     * Map that allows to get registered item by name
+     */
+    static final HashMap<String, Item> itemMap = new HashMap<String, Item>();
```

Comment:
caching `Item`s in Bindings is never a good idea. Afaics you don't really need the Item functionality right? Can you please explain why you are going to cache items here?